Clicker/ViewControllers/PollsDateViewController+Extension.swift

```diff
@@ -171,8 +172,12 @@ extension PollsDateViewController: SocketDelegate {
     }
     
     func pollStarted(_ poll: Poll, userRole: UserRole) {
-        if let lastPollsDateModel = pollsDateArray.first, lastPollsDateModel.polls.contains(where: { $0.id == poll.id })
-                || lastPollsDateModel.polls.first?.state == .live { return }
+        if let lastPollsDateModel = pollsDateArray.first, let id = poll.id {
+            if lastPollsDateModel.polls.contains(where: { otherPoll -> Bool in
```

Comment:
i feel like this stuff should be in a `guard let`

Reply:
yeah you could prob just chain it with the last one